gke/grpc-bookstore.yaml

```diff
@@ -49,7 +49,7 @@ spec:
           "--listener_port=9000",
           "--service=SERVICE_NAME",
           "--rollout_strategy=managed",
-          "--backend=grpc://127.0.0.1:8000"
+          "--backend=grpc://127.0.0.1:8000",
```

Comment:
This trailing period looks not needed to me? 
https://symfony.com/doc/current/components/yaml/yaml_format.html

Reply:
Are you sure?  I was using this yaml [validator](https://codebeautify.org/yaml-validator). It did not pass.  I will use kubectl to test it

Reply:
You are correct.  Just verified with "kubectr",  trailing peroid at that last argument is not needed.